conf/booksample/chapter-05.adoc

```diff
@@ -1,48 +1,17 @@
-== Using Mathematical Formulas
```

Comment:
It just looks as if - I tought it would be nice to have the docu for ditaa near the plantuml docu and therefor moved the chapters. I changed this in the new version and added the ditaa docu as `chapter-08.adoc`.